pkg/artifactory/resource_artifactory_local_cargo_repository.go

```diff
@@ -0,0 +1,42 @@
+package artifactory
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceArtifactoryLocalCargoRepository() *schema.Resource {
+	const packageType = "cargo"
+
+	var cargoLocalSchema = mergeSchema(baseLocalRepoSchema, map[string]*schema.Schema{
+		"anonymous_access": {
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Default:     false,
+			Description: `(Optional) Cargo client does not send credentials when performing download and search for crates. Enable this to allow anonymous access to these resources (only), note that this will override the security anonymous access option. Default value is 'false'.`,
+		},
+	}, repoLayoutRefSchema("local", packageType), compressionFormats)
+
+	type CargoLocalRepo struct {
+		LocalRepositoryBaseParams
+		AnonymousAccess bool `hcl:"anonymous_access" json:"cargoAnonymousAccess"`
```

Comment:
You don't need the `hcl:` tag here anymore now that the key and hcl attribute matches.